src/crypto/mod.rs

```diff
@@ -19,6 +22,11 @@ mod srtp;
 pub use srtp::{aead_aes_128_gcm, aes_128_cm_sha1_80, new_aead_aes_128_gcm};
 pub use srtp::{new_aes_128_cm_sha1_80, srtp_aes_128_ecb_round, SrtpProfile};
 
+#[cfg(all(feature = "openssl", feature = "wincrypto"))]
+compile_error!("features `openssl` and `wincrypto` are mutually exclusive");
+#[cfg(not(any(feature = "openssl", feature = "wincrypto")))]
+compile_error!("either `openssl` or `wincrypto` must be enabled");
```

Comment:
I am not quite sure why it works if we depend on `str0m` with `default-features = false, features = ["sha1"]` but I expected this line to cause problems.

In the str0m repo, `cargo check --no-default-features --features sha1` doesn't work.